ci/tests/general/general_disk.py

```diff
@@ -196,6 +199,55 @@ def partition_disk(disk):
         return disk.partitions[0]
 
     @staticmethod
+    def unpartition_disk(disk=None, partitions=None, wait=True):
+        """
+        Return disk to RAW state
+        :param disk: Disk DAL object
+        :param partitions: Partitions DAL object list
+        :return: None
+        """
+        if disk is None and partitions is None:
+            return
+        if disk is None:
+            disk = partitions[0].disk
+        if partitions is None:
+            partitions = disk.partitions
+        if len(disk.partitions) == 0:
+            return
+
+        root_client = SSHClient(disk.storagerouter, username='root')
+        for partition in partitions:
+            GeneralDisk.unmount_partition(root_client, partition)
+        root_client.run("parted -s /dev/{0} mklabel gpt".format(disk.name))
+        GeneralStorageRouter.sync_with_reality(disk.storagerouter)
+        counter = 0
+        timeout = 60
+        while counter < timeout:
+            time.sleep(1)
+            disk = GeneralDisk.get_disk(guid=disk.guid)
+            if len(disk.partitions) == 0:
+                break
+            counter += 1
+        if counter == timeout:
+            raise RuntimeError('Removing partitions failed for disk:\n {0} '.format(disk.name))
+
+    @staticmethod
+    def unmount_partition(root_client, partition):
+        """
+        Unmount a partition
+        :param root_client: ssh-connection
+        :param partition: Partition Dal object
+        :return: None
+        """
+        mounted = False
```

Comment:
And i think we already have a library method to check if a mountpoint is actually mounted
Please grep on 'mount' in library files to verify for sure